### PR TITLE
HowTo PubSub with multiple namespaces.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

HowTo explaining using PubSub with multiple namespaces.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #528 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [X] Extended the documentation

### CODE REVIEW

File: howto/pubsub-namespaces/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+```
+kubectl get pod # Copy POD names and use in the next commands.
+```
+
+Display logs:
+```
+kubectl logs node-subscriber-XYZ node-subscriber
+kubectl logs python-subscriber-XYZ python-subscriber
+```
+
```
Comment (orizohar) [inline howto/pubsub-namespaces/README.md:93]: If everything went well, what should users see here?

File: howto/pubsub-namespaces/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+kubectl create namespace namespace-a
+kubectl config set-context --current --namespace=namespace-a
+```
+
+Install Redis (master and slave) on `namespace-a`:
+```
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update
+helm install redis bitnami/redis
+export REDIS_PASSWORD=$(kubectl get secret --namespace namespace-a redis -o jsonpath="{.data.redis-password}" | base64 --decode)
```
[the remark this answers is not in the document]
Comment (artursouza) [reply howto/pubsub-namespaces/README.md:34]: I refer to the source of truth instead. Done.

File: howto/pubsub-namespaces/README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+kubectl config set-context --current --namespace=namespace-b
+```
+
+Deploy resources to `namespace-b`, including the Redis component:
+```
+kubectl apply -f deploy/redis.yaml
+kubectl apply -f deploy/react-form.yaml
+```
+
+Now, find the IP address for react-form, open it on your browser and publish messages to each topic (A, B and C).
```
[the remark this answers is not in the document]
Comment (artursouza) [reply howto/pubsub-namespaces/README.md:74]: I had it but forgot to add it. done.

File: howto/pubsub-namespaces/README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+kubectl delete -f deploy/node-subscriber.yaml  --namespace namespace-a
+kubectl delete -f deploy/python-subscriber.yaml  --namespace namespace-a
+kubectl delete -f deploy/react-form.yaml  --namespace namespace-b
+kubectl delete -f deploy/redis.yaml  --namespace namespace-b
+kubectl config set-context --current --namespace=default
+kubectl delete namespace namespace-a
+kubectl delete namespace namespace-b
+```
+
+Thanks.
```
[the remark this answers is not in the document]
Comment (artursouza) [reply howto/pubsub-namespaces/README.md:107]: :( Done.

File: howto/pubsub-namespaces/README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+| Redis slave | X ||
+| Dapr's PubSub component | X | X |
+| Node subscriber | X ||
+| Python subscriber | X ||
+| React UI publisher | | X|
+
+## Pre-requisites
+
+* [Dapr installed](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md.) on any namespace since Dapr works at the cluster level.
+
```
[the remark this answers is not in the document]
Comment (artursouza) [reply howto/pubsub-namespaces/README.md:20]: Done.

File: howto/pubsub-namespaces/README.md (hunk 1 above)
Comment (artursouza) [reply howto/pubsub-namespaces/README.md:91]: Described output similar to how the sample does.

**Pull request merged**